src/main/java/BlueTurtle/TSE/JSONFormatter.java

```diff
@@ -33,9 +34,22 @@
 	 */
 	public void format() throws IOException {
 		List<Warning> totalWarnings = new ArrayList<Warning>();
-		totalWarnings.addAll(parseCheckStyleXML());
-		totalWarnings.addAll(parsePMDXML());
-		totalWarnings.addAll(parseFindBugsXML());
+		
+		File file = new File(JavaController.getCheckStyleOutputFile());
+		if(file.exists() && !file.isDirectory()) { 
```

Comment:
Isn't it better to throw this in a function?, because it is doing the same thing for each of the output file.